plugins/basketball-scoreboard/ncaam_basketball_managers.py

```
@@ -88,34 +88,57 @@ def _get_team_id(self, team_abbr: str) -> Optional[str]:
+
+            # Cache the team's record summary separately (1-hour TTL via max_age on read)
+            team_record_summary = data.get("team", {}).get("recordSummary", "")
+            if team_record_summary:
+                self.cache_manager.set(record_summary_key, team_record_summary)
+
+            # Enrich the in-memory copy before returning
+            self._enrich_events_with_records(events, team_id, team_record_summary)
+
             self.logger.info(f"Fetched {len(events)} events for team {team_id} season {season_year}")
-            return result
+            return {"events": events}
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)